tachyon/zk/plonk/circuit/examples/BUILD.bazel

```diff
@@ -21,7 +21,9 @@ tachyon_cc_library(
     deps = ["//tachyon/zk/plonk/circuit"],
 )
 
-tachyon_cc_unittest(
+# TODO(dongchangYoo): Change |tachyon_cc_test| to
+# |tachyon_cc_unittest| after increasing the timeout for unittests.
```

Comment:
This comment doesn't make sense. You need to say in this way

```suggestion
# TODO(dongchangYoo): This is failed in CI because of timeout, 60 secs. 
# Change |tachyon_cc_test| to |tachyon_cc_unittest| once CI can run within the timeout.
```

Reply:
I mistakenly thought that the timeout would eventually be extended. 
I have edited the comment.